bin/tass

```diff
@@ -37,10 +36,31 @@ when 'create'
 
   config, aws_env, user_data = Tapjoy::AutoscalingBootstrap::Base.new.configure_environment(opts[:filename], opts[:env], opts[:config_dir])
   Aws.config[:region] = config[:aws_region]
-  confirm_config(aws_env, config, opts)
 
   Tapjoy::AutoscalingBootstrap::Base.new.check_clobber(opts, config)
+  unless confirm_config(aws_env, config, opts)
+    abort('Cannot continue if configuration is not correct.  Please fix.')
+  end
   Tapjoy::AutoscalingBootstrap::AutoscalingGroup.new.create(opts, config, aws_env, user_data)
+
+  if config.key?(:elb)
+    config[:elb].each do |elb|
+      elb.each do |elb_name, elb_config|
+        Tapjoy::AutoscalingBootstrap.elb_name = elb_name
+        Tapjoy::AutoscalingBootstrap.create_elb = true
```

Comment:
What if you just want to attach to existing elbs, not create + attach?

How does this design account for that?